File: 01924.cpp
#include <iostream>
using namespace std;

int main() {
	int days[11] = { 31, 28, 31, 30, 31, 30, 31, 31, 30, 31, 30 };
	int x, y; cin >> x >> y;

	int day = 0;
	for (int i = 1; i < x; i++)
		day += days[i - 1];
	day += y - 1;

	switch (day % 7) {
	case 0:
		cout << "MON\n";
		break;

	case 1:
		cout << "TUE\n";
		break;

	case 2:
		cout << "WED\n";
		break;

	case 3:
		cout << "THU\n";
		break;
		
	case 4:
		cout << "FRI\n";
		break;

	case 5:
		cout << "SAT\n";
		break;

	case 6:
		cout << "SUN\n";
		break;
	}
}

File: 01924.py
days = [31, 28, 31, 30, 31, 30, 31, 31, 30, 31, 30]
x, y = map(int, input().split())

day = 0
for i in range(1, x):
    day += days[i - 1]
day += y - 1

week = day % 7
if week == 0:
    print("MON")
elif week == 1:
    print("TUE")
elif week == 2:
    print("WED")
elif week == 3:
    print("THU")
elif week == 4:
    print("FRI")
elif week == 5:
    print("SAT")
else:
    print("SUN")
